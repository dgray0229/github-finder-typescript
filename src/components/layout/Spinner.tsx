import React, { Fragment } from 'react'
import Loading from './spinner.gif'

const Spinner: React.SFC = () => (
    <Fragment>
        <img src={Loading} alt="Loading..." style={{width: '200px', margin: 'auto', display: 'block'}} />
    </Fragment>
)

export default Spinner