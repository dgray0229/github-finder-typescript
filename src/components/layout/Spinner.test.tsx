import React from 'react';
import Enzyme, { shallow } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';

import Spinner from './Spinner';

Enzyme.configure({ adapter: new Adapter() })

describe("Spinner Component", () => {
  test("renders main component", () => {
    const wrapper = shallow(<Spinner />)
    expect(wrapper.exists()).toBe(true)
    expect(wrapper).toMatchSnapshot();

  })
})
