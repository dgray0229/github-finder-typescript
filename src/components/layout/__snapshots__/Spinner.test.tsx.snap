// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Spinner Component renders main component 1`] = `
<Fragment>
  <img
    alt="Loading..."
    src=""
    style={
      Object {
        "display": "block",
        "margin": "auto",
        "width": "200px",
      }
    }
  />
</Fragment>
`;
